### PR TITLE
support nodejs 18 and enable next loader in the chain to continue resolution

### PR DESCRIPTION
**Pull request by giltayar**

### CODE REVIEW

**Review by joeldenning — APPROVED** [review]
Thank you

**Review by joeldenning — COMMENTED** [review]
The package.json change broke this. I will undo it.

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "husky": "^4.3.0",
     "left-pad": "^1.3.0",
     "mocha": "^8.1.3",
-    "prettier": "^2.1.2",
+    "prettier": "^2.4.1",
```
Comment (joeldenning) [inline package.json:40]:
```suggestion
    "prettier": "^2.1.2",
```

**Comment by joeldenning** [comment]
Released in https://github.com/node-loader/node-loader-import-maps/releases/tag/v1.1.0

**Comment by raihle** [comment]
FYI (and perhaps for future searchers), this fix broke our (admittedly broken) config where `@node-loader/http` was _before_ `@node-loader/import-maps` in the chain. Reversing the order fixed the problem.

Example error message:

> Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. Received protocol 'https:'

**Pull request merged**